src/Avalonia.Themes.Fluent/Controls/SplitButton.xaml

```diff
@@ -0,0 +1,205 @@
+                    BorderBrush="{TemplateBinding BorderBrush}"
+                    Content="{TemplateBinding Content}"
+                    ContentTemplate="{TemplateBinding ContentTemplate}"
+                    Command="{TemplateBinding Command}"
+                    CommandParameter="{TemplateBinding CommandParameter}"
+                    FontFamily="{TemplateBinding FontFamily}"
+                    FontSize="{TemplateBinding FontSize}"
+                    FontWeight="{TemplateBinding FontWeight}"
+                    HorizontalAlignment="Stretch"
+                    VerticalAlignment="Stretch"
+                    HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
+                    VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+                    Padding="{TemplateBinding Padding}"
+                    Focusable="False"
+                    KeyboardNavigation.IsTabStop="False" />
+
+            <Button x:Name="PART_SecondaryButton"
+                    Grid.Column="1"
+                    MinWidth="{DynamicResource SplitButtonSecondaryButtonSize}"
+                    BorderBrush="{TemplateBinding BorderBrush}"
+                    Foreground="{TemplateBinding Foreground}"
+                    Background="{TemplateBinding Background}"
+                    BorderThickness="{TemplateBinding BorderThickness}"
+                    HorizontalContentAlignment="Right"
+                    VerticalContentAlignment="Center"
+                    HorizontalAlignment="Stretch"
+                    VerticalAlignment="Stretch"
+                    Focusable="False"
+                    KeyboardNavigation.IsTabStop="False" />
+          </Grid>
+        </ControlTemplate>
+      </Setter.Value>
+    </Setter>
+  </Style>
+
+  <!-- Default overridable styles -->
+  <Style Selector="SplitButton /template/ Button#PART_PrimaryButton">
+    <Setter Property="CornerRadius" Value="{TemplateBinding CornerRadius, Converter={StaticResource LeftCornerRadiusFilterConverter}}" />
+  </Style>
+  <Style Selector="SplitButton /template/ Button#PART_SecondaryButton">
+    <Setter Property="CornerRadius" Value="{TemplateBinding CornerRadius, Converter={StaticResource RightCornerRadiusFilterConverter}}" />
+    <Setter Property="Padding" Value="9,0" />
+    <Setter Property="Content">
+      <Template>
+        <PathIcon Height="12"
+                  Width="12"
+                  Data="{StaticResource SplitButtonSecondaryButtonIconGeometry}" />
+      </Template>
+    </Setter>
+  </Style>
+
+  <!-- Primary and Secondary buttons PointerOver State -->
+  <Style Selector="SplitButton /template/ Button#PART_PrimaryButton:pointerover /template/ ContentPresenter,
+         SplitButton /template/ Button#PART_SecondaryButton:pointerover /template/ ContentPresenter">
+    <Setter Property="Background" Value="{DynamicResource SplitButtonBackgroundPointerOver}" />
```

Comment:
Didn't know you could have multiple target controls specified in the selector like this. That is good to know.

Reply:
Yes. It's the main simplification, which makes additional pseudoclasses not needed.

Reply:
Note that you can have state pseudoclasses on multiple levels of the selector (on the SplitButton and on the Button), which also makes it even more flexible. It's not free performance wise, of course, but isn't much more expensive than separated pseudoclass and logic behind it.

Reply:
Yes, I notice how you were using :pointerover in the PrimaryButton directly. That is a nice feature.